pages/Instalacao Windows.md

```diff
@@ -0,0 +1,134 @@
+<p align="center">
+    <img src="../assets/header-4noobs.svg">
+</p>
+
+<h1 align="center">Containers4Noobs</h1>
+
+<p align="center">
+    <img src="../assets/containers.jpg" />
+</p>
+
+# Instalação do Ambiente no Windows
+
+Essa página tem como objetivo auxiliar na instalação de ferramentas de Containers no OS Windows, sendo que nesse tutorial abordaremos duas ferramentas:
+
+- 👍 Podman
+- Docker
+
+# Instalando a WSL 2 e Requisitos
+
+Os ambientes de visualização de containers (em sua grande maioria) são baseados em sistemas Unix (Linux), por isso existe uma incompatibiliade nativa entre os containers e o Windows.
+
+Para solucionar isso, o OS disponibiliza uma ferramenta chamada WSL (Windows Subsystem For Linux).
+
+<p align="center">
```

Comment:
Licença da imagem é compatível com o repositório? Se não for sua, é bom deixar pelo menos os créditos da imagem, com link para a fonte.

Reply:
```
<p>
    <img src="path_to_image" alt>
    <em>image_caption</em>
</p>
```

Assim para colocar os créditos e tal no lugar de `image_caption`